pbj-core/pbj-compiler/src/main/java/com/hedera/pbj/compiler/impl/Common.java

```diff
@@ -1,203 +1,688 @@
 public final class Common {
-	/** The indent for fields, default 4 spaces */
-	public static final String FIELD_INDENT = " ".repeat(4);
-
-	/** Number of bits used to represent the tag type */
-	static final int TAG_TYPE_BITS = 3;
-
-	/** Wire format code for var int */
-	public static final int TYPE_VARINT = 0;
-	/** Wire format code for fixed 64bit number */
-	public static final int TYPE_FIXED64 = 1;
-	/** Wire format code for length delimited, all the complex types */
-	public static final int TYPE_LENGTH_DELIMITED = 2;
-	/** Wire format code for fixed 32bit number */
-	public static final int TYPE_FIXED32 = 5;
-
-
-	/**
-	 * Makes a tag value given a field number and wire type.
-	 *
-	 * @param wireType the wire type part of tag
-	 * @param fieldNumber the field number part of tag
-	 * @return packed encoded tag
-	 */
-	public static int getTag(final int wireType, final int fieldNumber) {
-		return (fieldNumber << TAG_TYPE_BITS) | wireType;
-	}
-
-	/**
-	 * Make sure first character of a string is upper case
-	 *
-	 * @param name string input who's first character can be upper or lower case
-	 * @return name with first character converted to upper case
-	 */
-	public static String capitalizeFirstLetter(String name) {
-		if (name.length() > 0) {
-			if (name.chars().allMatch(Character::isUpperCase)) {
-				return Character.toUpperCase(name.charAt(0)) + name.substring(1).toLowerCase();
-			} else {
-				return Character.toUpperCase(name.charAt(0)) + name.substring(1);
+    /** The indent for fields, default 4 spaces */
+    public static final String FIELD_INDENT = " ".repeat(4);
+
+    /** Number of bits used to represent the tag type */
+    static final int TAG_TYPE_BITS = 3;
+
+    /** Wire format code for var int */
+    public static final int TYPE_VARINT = 0;
+    /** Wire format code for fixed 64bit number */
+    public static final int TYPE_FIXED64 = 1;
+    /** Wire format code for length delimited, all the complex types */
+    public static final int TYPE_LENGTH_DELIMITED = 2;
+    /** Wire format code for fixed 32bit number */
+    public static final int TYPE_FIXED32 = 5;
+
+    /**
+     * Makes a tag value given a field number and wire type.
+     *
+     * @param wireType the wire type part of tag
+     * @param fieldNumber the field number part of tag
+     * @return packed encoded tag
+     */
+    public static int getTag(final int wireType, final int fieldNumber) {
+        return (fieldNumber << TAG_TYPE_BITS) | wireType;
+    }
+
+    /**
+     * Make sure first character of a string is upper case
+     *
+     * @param name string input who's first character can be upper or lower case
+     * @return name with first character converted to upper case
+     */
+    public static String capitalizeFirstLetter(String name) {
+        if (name.length() > 0) {
+            if (name.chars().allMatch(Character::isUpperCase)) {
+                return Character.toUpperCase(name.charAt(0)) + name.substring(1).toLowerCase();
```

Comment:
The first char is already capitalized, no need to call toUpperCase() for it again

Reply:
This is the existing code. It got modified by the spotless formatter, but the code is the same as it was before this PR.